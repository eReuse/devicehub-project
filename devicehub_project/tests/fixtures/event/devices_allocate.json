{
  "@type": "devices:Allocate",
  "date": "2016-07-25T18:18:13",
  "description": "I allocate these devices..."
}