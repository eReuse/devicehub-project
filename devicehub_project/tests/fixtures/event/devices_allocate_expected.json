{
  "@type": "devices:Allocate",
  "date": "2016-07-25T18:18:13",
  "description": "I allocate these devices...",
  "project": "https://www.reutilitza.cat/projects/xyz",
  "secured": false,
  "incidence": false
}